gstreamer-rtsp/src/auto/r_t_s_p_extension.rs

```diff
@@ -0,0 +1,93 @@
+// This file was generated by gir (37f7805+) from gir-files (???)
+// DO NOT EDIT
+
+use RTSPResult;
+use RTSPUrl;
+use ffi;
+use glib::object::IsA;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use gst;
+use std::mem;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct RTSPExtension(Object<ffi::GstRTSPExtension, ffi::GstRTSPExtensionInterface>);
+
+    match fn {
+        get_type => || ffi::gst_rtsp_extension_get_type(),
+    }
+}
+
+unsafe impl Send for RTSPExtension {}
+unsafe impl Sync for RTSPExtension {}
```

Comment:
I think we can just not generate GstRTSPExtension for now

Reply:
done